docker-compose.yml

```diff
@@ -27,7 +27,8 @@ services:
     build:
       context: .
       args:
-        DEVEL: "yes"
+        DEVEL: "no"
```

Comment:
Docker Compose is only used locally, so setting DEVEL=yes is always the right call here :)

Let's revert the changes to DEVEL args in docker-compose.yml and this is good to go!